src/lib.rs

```diff
@@ -477,4 +458,45 @@ mod tests {
         assert_ne!(normalize_word("cent"), normalize_word("chant"));
         assert_ne!(normalize_word("cough"), normalize_word("cow"));
     }
+
+    #[test]
+    fn multibyte_chars() {
+        assert_eq!(normalize_word("piétro"), "piétro");
+        assert_eq!(normalize_word("piéitly"), "piéatly");
+    }
+
+    #[test]
+    fn short_words() {
+        assert_eq!(normalize_word("a"), "a");
+        assert_eq!(normalize_word("be"), "be");
+        assert_eq!(normalize_word("at"), "at");
+        assert_eq!(normalize_word("do"), "do");
+    }
+
+    #[test]
+    fn replace_end_if_tests() {
```

Comment:
Having a test that tries to search for something even longer than the passed in word would be a good test to have. Other than that' it looks good.

Reply:
Good thought, thanks! Done.